### PR TITLE
Deprecate tokens for the old design language, upgrade dependencies

### PR DESCRIPTION
**Pull request by kaelig**
- Updates dependencies and Node.js (this PR was originally going to be a bucket for all v3-related updates… happy to remove this change from this PR if that's too much)
- Removes all the code generating tokens for the old design language
- Freezes the `dist` folder, which will be enforced thanks to this Caution Tape rule: https://github.com/Shopify/spy/pull/8784

For context, the `dist` folder was initially versioned to make reviewing PRs significantly easier (reviewers can check the generated output at a glance, also meaning we could get away with a very light test suite).

As of this PR, the new reason we’re versioning `dist` is so we can delete tech debt of scripts outputting old token files, without introducing breaking changes for consumers.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -17,6 +17,6 @@
     "**/.DS_Store": true,
     "**/.git": true,
     "**/node_modules": true,
-    "{build,dist-modern}/": true
+    "{build,dist-modern,dist}/": true
```
Comment (kaelig) [inline .vscode/settings.json:20]: Prevents accidental search/replace changes.

File: docs/index.html [hunk 2]
```diff
@@ -1,709 +1,871 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Design Tokens (legacy)</title>
```
Comment (kaelig) [inline docs/index.html:6]: This only changes the title and h1 – saving reformatted the entire file 💅

File: app.json [hunk 3]
```diff
@@ -1,20 +0,0 @@
-{
```
Comment (kaelig) [inline app.json:1]: We don't need to deploy deprecated/frozen docs to Heroku anymore, so this can probably go.

**Comment by BPScott** [comment]
Would love to know more about what you're plotting. Looking forward to chatting on Friday!

**Review by alex-page — APPROVED** [review]
@kaelig this is looking great, is there anything you need to ship this?

**Comment by kaelig** [comment]
> is there anything you need to ship this?

All I needed was an approval, so let's ship it :)

**Pull request merged**